src/views/shared/EditCreateCourseForm.vue

```diff
@@ -87,7 +87,7 @@
 
 <script lang="ts">
     import Router from "@/use/router/";
-    import { useStore } from "@/use/store/store";
+    import { store, useStore } from "@/use/store/store";
```

Comment:
```suggestion
    import { useStore } from "@/use/store/store";
```

did not test if this breaks things, but we should only access the store with getters anyway